<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="lwObjects">
        <property name="font">
         <font>
          <family>Ubuntu Nerd Font</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pbGetAll">
        <property name="text">
         <string>Всё</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGetDist">
        <property name="text">
         <string>по Дистанции</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGetTime">
        <property name="text">
         <string>по Врмени</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGetLetter">
        <property name="text">
         <string>по Буквам</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGetType">
        <property name="text">
         <string>по Типу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbTypeCount"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
